packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -132,7 +163,7 @@ class FlatTableState<T> extends State<FlatTable<T>>
   }
 
   void _initData() {
-    data = List.from(widget.data);
+    _originalData = UnmodifiableListView(List.from(widget.data));
```

Comment:
do we still need to initialize `data` here?

Reply:
No, since we always re-initialize it in `sortData` anyway.